deps/graph-parser/src/logseq/graph_parser/util.cljs

```diff
@@ -80,6 +80,7 @@
   (and (string? v) (>= (count v) 2) (= "\"" (first v) (last v))))
 
 (defn url?
+  "BUG: can not handle mailto: links, use this with caution"
```

Comment:
Don't know if this is a bug. Would be worth checking that features that use `gp-util/url?` support mailto links

Reply:
I've changed the doc comment. Leave a note on the usage case only.